offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/SelectItemsView.groovy

```diff
@@ -374,6 +374,7 @@ class SelectItemsView extends VerticalLayout implements Resettable {
 
             //specify size of grid and layout
             grid.setWidthFull()
+            descriptionColumn.setWidth(400)
```

Comment:
since we configure a central layout setting, it makes sense to define a constant in the class and reference it here.